.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "matrix-doc"]
```

Comment:
I don't think a git submodule is the right way to approach including the documentation for testing, but I don't know what the "right way" is off the top of my head.